collector/test/UtilityTest.cpp

```diff
@@ -117,15 +117,20 @@ TEST(ExtractContainerIDFromCgroupTest, TestExtractContainerIDFromCgroup) {
 }
 
 TEST(SanitizeUTF8Test, TestSanitizeUTF8_Invalid) {
-  std::string input(
-      "ab\x80"
-      "cd");
-  std::string expected_output("ab?cd");
+  using test_case = std::pair<std::string, std::string>;
+  std::vector<test_case> tests = {
+      {"ab\200cd", "ab?cd"},
+      {"ab\200", "ab?"},
+      {"\200ab\200", "?ab?"},
+      {"\200\200ab\200", "??ab?"},
+  };
```

Comment:
[nit] I suggest that we keep a valid ascii-only string and a valid string with expanded codes in the test.

Reply:
There already is a separate test for a valid ASCII string, I added a second case to that one with a text in Japanase, hopefully that is good enough.